src/test/jetty/jetty-env.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<Configure class="org.eclipse.jetty.webapp.WebAppContext">
+
+    <New id="logback" class="org.eclipse.jetty.plus.jndi.EnvEntry">
+        <Arg>java-webapp-logback-config</Arg>
```

Comment:
You will want the value to match the name of the log configuration. i.e.: licensemodule-logback-config

Reply:
Fixed, also added the property file as env variable. Added template property to jetty folder